tests/test_assessment.py

```diff
@@ -9030,6 +9030,60 @@ def test_can_submit_foreign_language_text_response(self):
         self.assertTrue(data['correct'])
         self.assertIn('<p/>', data['feedback'])
 
+    def test_can_edit_max_strings(self):
+        item = self.create_item()
+        old_max_strings = item['question']['maxStrings']
```

Comment:
Yeah, but you would then also have to change the other tests that expect the 300 to be there...since the same default item is used for other tests.